index.html

```diff
@@ -11,6 +12,85 @@
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+
+    <div class="container">
+
+
+      <header>
+
+          <nav class = "navmenu">
+              <a class="nav-services" href = "services.html">Services</a>
+              <a href = "#">Product</a>
+              <a href = "#">Vision</a>
+              <a href = "#">Features</a>
+              <a href = "#">About</a>
+              <a href = "#">Contact</a>          
+          </nav>
+
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      
+      </header>
+
+      <section class="jumbo">
+
+        <div class="jumbo-text">
+            <h1>Innovation<br> On<br> Demand</h1>
+            <button>Get Started</button>
+        </div>
+
+        <img class="logo" src="img/header-img.png" alt="Great Idea! Header">
+
+      </section>
+
+      <section class="top-content">
+
+          <div class="col-text">
+              <h4>Features</h4>
+              <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+
+          <div class="col-text">
+              <h4>About</h4>
+              <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>    
+          </div>
+
+      </section>
+
+      <img class="logo" src="img/mid-page-accent.jpg" alt="Great Idea! Mid-page">
+
+      <section class="bottom-content">
+
+          <div class="bc-text">
+              <h4>Services</h4>
+              <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+
+          <div class="bc-text">
+              <h4>Product</h4>
+              <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>   
+          </div>
+
+          <div class="bc-text">
+              <h4>Vision</h4>
+              <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>    
+          </div>
+
+      </section>
+
+      <footer>
+          <h4>Contact</h4>
+
+          <p style="margin-bottom:20px">123 Way 456 Street<br>
```

Comment:
it's best to avoid using in-line styles directly in the HTML code. anything declared here will outweigh the settings in your CSS file and forces you to find your styling in two separate locations which can be confusing.